src/renderer/pages/torrent-list-page.js

```diff
@@ -278,10 +279,17 @@ module.exports = class TorrentList extends React.Component {
       )
     } else {
       // We do know the files. List them and show download stats for each one
-      const fileRows = torrentSummary.files
+      const sortByName = this.props.state.saved.prefs.sortByName
```

Comment:
The code with javascript's native `localeCompare` could be something like this:
```
const sortByName = this.props.state.saved.prefs.sortByName
const collator = new Intl.Collator(undefined, { numeric: true, sensitivity: 'base' })
let fileRows = torrentSummary.files
    .filter((file) => !file.path.includes('/.____padding_file/'))
    .map((file, index) => ({ file, index }))

if (sortByName) {
    fileRows = fileRows.sort((a, b) => collator.compare(a.file.name, b.file.name))
}

fileRows = fileRows.map((obj) => this.renderFileRow(torrentSummary, obj.file, obj.index))
```

Reply:
Thanks. I had kept the same logic as the previous commit this PR was extended from, but agree this would be easier to keep deps from bloating unnecessarily.